api/v1beta1/openstackcontrolplane_types.go

```diff
@@ -37,6 +37,14 @@ type OpenStackControlPlaneSpec struct {
 	// +kubebuilder:default={ctlplane,external}
 	// OpenStackClientNetworks the name(s) of the OpenStackClientNetworks used to attach the openstackclient to
 	OpenStackClientNetworks []string `json:"openStackClientNetworks"`
+
+	// +kubebuilder:validation:Optional
+	// PhysNetworks - physical networks list with optional MAC address prefix, used to create static OVN Bridge MAC address mappings.
+	// Unique OVN bridge mac address per node is dynamically allocated by creating OpenStackMACAddress resource and create a MAC per physnet per node.
```

Comment:
Small nit due to potential ambiguity: I wonder if users might confuse "node" here with an OCP node instead of an OSP node?